src/Components/TimeSheets/Form/timesheetForm.js

```diff
@@ -0,0 +1,206 @@
+      .then((res) => res.json())
+      .then((response) => {
+        setTasks(response.data);
+      });
+  }, []);
+
+  useEffect(async () => {
+    const urlForm = new URLSearchParams(window.location.search);
+    const id = urlForm.get('id');
+    if (id) {
+      setFormPut(false);
+      const urlFetch = await fetch(`${process.env.REACT_APP_API_URL}/time-sheets/${id}`, {
+        method: 'GET'
+      });
+      const response = await urlFetch.json();
+
+      setTimesheetAdded({
+        description: response.data.description,
+        date: response.data.date,
+        hours: response.data.hours,
+        project: response.data.project._id,
+        employee: response.data.employee._id,
+        task: response.data.task._id
+      });
+    } else {
+      return null;
+    }
+  }, []);
+
+  const onChange = (event) => {
+    setTimesheetAdded({ ...timesheetAdded, [event.target.name]: event.target.value });
+    if (event.target.name === 'project') {
+      const selectedProject = projects.find((project) => project._id === event.target.value);
+      const projectEmployees = selectedProject.employees.map((employee) => employee.employee);
+      setEmployees(projectEmployees);
+    }
+  };
+
+  const onSubmit = async (event) => {
+    if (formPut) {
+      event.preventDefault();
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/time-sheets`, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          description: timesheetAdded.description,
+          date: timesheetAdded.date,
+          hours: timesheetAdded.hours,
+          project: timesheetAdded.project,
+          employee: timesheetAdded.employee,
+          task: timesheetAdded.task
+        })
+      });
+      const data = await response.json();
+      if (!data.error) {
+        window.location.assign('/time-sheets');
+      } else {
+        setErrorState(data.message);
+      }
+    } else {
+      const urlForm = new URLSearchParams(window.location.search);
+      const id = urlForm.get('id');
+      event.preventDefault();
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/time-sheets/${id}`, {
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          description: timesheetAdded.description,
+          date: timesheetAdded.date,
+          hours: timesheetAdded.hours,
+          project: timesheetAdded.project,
+          employee: timesheetAdded.employee,
+          task: timesheetAdded.task
+        })
+      });
+      const data = await response.json();
+      if (!data.error) {
+        window.location.assign('/time-sheets');
+      } else {
+        setErrorState(data.message);
+      }
+    }
+  };
+  return (
+    <div>
+      <form onSubmit={onSubmit} className={styles.container}>
+        {formPut ? <h2>Create a Timesheet</h2> : <h2>Edit a Timesheet</h2>}
+        <h3>{errorState}</h3>
```

Comment:
```suggestion
        {errorState  && <h3>{errorState}</h3>}
```
In this way we are not rendering an empty h3